src/resources/items/armor.ts

```diff
@@ -70,5 +70,16 @@ export const armor = armorObject({
 		level: 2,
 		slotsUsed: 2,
 		itemLevel: 3
-	}
+	},
+	tactical_bodyarmor: {
+		type: 'Body Armor',
+		name: 'Tactical_Bodyarmor',
```

Comment:
think tactical armor is shorter, easier to remember. also make sure the name property is written the same as the above (uppercase matters)
```suggestion
	tactical_armor: {
		type: 'Body Armor',
		name: 'tactical_armor',
```